src/modules/routes/Routes.ts

```diff
@@ -20,8 +20,8 @@ export default class Routes {
         return this.regionRoutes.find((routeData) => routeData.region === region && routeData.number === route);
     }
 
-    public static getRoutesByRegion(region: GameConstants.Region): RegionRoute[] {
-        return this.regionRoutes.filter((routeData) => routeData.region === region);
+    public static getRoutesByRegion(region: GameConstants.Region, ignoreIgnoredRouteInCalculations = false): RegionRoute[] {
+        return this.regionRoutes.filter((routeData) => routeData.region === region && !(ignoreIgnoredRouteInCalculations && routeData.ignoreRouteInCalculations));
```

Comment:
For now I think we should keep this function as-is and just have the extra filter in `wandererToRoute`, it isn't called often enough to have any impact on performance and this new override isn't used anywhere else at this time.

Reply:
Addressed.